src/audit_plugins/focus_indicator_audit.py

```diff
@@ -120,6 +121,20 @@ def screenshot(self) -> Any:
             cv2.IMREAD_COLOR,
         )
 
+    def __find_main_content_element(self) -> Any | None:
+        try:
+            main_element = self.browser.driver.find_element(By.TAG_NAME, "main")
+            logging.info("Checking focus indication within <main> element")
+            return main_element
+        except WebDriverException:
+            logging.warning("Failed to find main element, falling back to body")
+
+        try:
+            return self.browser.driver.find_element(By.TAG_NAME, "body")
+        except Exception:  # pylint: disable=broad-except
```

Comment:
I take it this can return an annoying variety of exceptions rather than just WebDriverException?

Reply:
I would hope that it only throws `WebDriverException`s for expected errors, but the existing code uses `Exception` which I'm preserving for now to play it safe since I've not used the tool in anger - for new code though I'm using more specific exceptions in part as a way to try teasing out confirmation that that is fine to switch to